spring-rest/src/main/java/org/baeldung/config/WebConfig.java

```diff
@@ -42,6 +43,7 @@ public void configureMessageConverters(final List<HttpMessageConverter<?>> messa
         // messageConverters.add(new MappingJackson2XmlHttpMessageConverter(builder.createXmlMapper(true).build()));
         messageConverters.add(new ProtobufHttpMessageConverter());
         messageConverters.add(new KryoHttpMessageConverter());
+        messageConverters.add(new ByteArrayHttpMessageConverter());
```

Comment:
It's connected with previous article, Top 7 mistakes, as in one example I am returning image itself, and this converts byte to browser friendly version)